integration-tests/suites/types/network.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+type NetworkInfo struct {
+	LocalAddress   string
+	RemoteAddress  string
+	Role           string
+	SocketFamily   string
+	CloseTimestamp string
+}
+
+func (n *NetworkInfo) IsActive() bool {
+	// no close timestamp means the connection is open, and active
+	return n.CloseTimestamp == "(timestamp: nil Timestamp)"
```

Comment:
There's a `nilTimeStamp` constant in `integration-tests/suites/base.go`, could somehow move it into this file and test against that directly?